introduction/motivation.md

```diff
@@ -62,13 +61,13 @@ products cope with JavaScript-heavy application frontends and REST APIs.
 Two years after its inception the Juice Shop was submitted and accepted
 as an _OWASP Tool Project_ by the
 [Open Web Application Security Project](https://owasp.org) in September
-2016\. This move increased the overall visibility and outreach of the
+2016. This move increased the overall visibility and outreach of the
```

Comment:
The escaped dot `\.` as intentional as otherwise the Markdown parser will treat this as the beginning of a numbered list.